apps/ledger-live-desktop/src/renderer/components/WebPlatformPlayer/TopBar.jsx

```diff
@@ -197,6 +206,13 @@ const WebPlatformTopBar = ({
     }
   }, [webviewRef]);
 
+  const onBackToLiveApp = useCallback(() => {
+    const webview = webviewRef.current;
+    if (webview) {
+      webview.loadURL(lastManifestAppURL.current);
```

Comment:
How is "loadURL" behaving with the navigation history ? Is it erasing the history ? Pushing a new entry ? This could affect the overall user experience.

Reply:
I believe this is pushing a new entry by default. @brunospolidorio can you check ?
If easily fixed, I think the cleanest is to erase the history, but @IAmMorrow keep in mind this feature is not supposed to be used together with browser navigation, it is a workaround as browser navigation is broken on many of our providers apps.

Reply:
Our team goal is to have navigation enabled for every single web3app soon so we can prepare for the web transition.